README.Rmd

```diff
@@ -62,11 +62,12 @@ WW_Delta
 
 ## Data types
 
-Three spatial data types are included in this package
+Four spatial data types are included in this package
 
 1) **Waterways**: Shapefiles of water coverage. These data files are preceded by the prefix *WW*.
 2) **Regions**: Shapefiles with sets of regions, subregions, or strata. These data files are preceded by the prefix *R*.
 3) **Habitat**: Shapefiles with habitat data. These data files are preceded by the prefix *H*.
+4) **Stations**: Shapefile with data on most IEP long-term monitoring programs.
```

Comment:
If you adopt my suggestion of using the P prefix for points, this would need to be changed to define that prefix for point datasets

Reply:
Done.